### ISSUE TITLE
Cannot put URL in SERVER_NAME or SERVER_DESCRIPTION in 0.19.0

### ISSUE THREAD

**Issue by bostrt** [thijsvanloef/palworld-server-docker#210]
## Describe the bug

Server name and description are not set when they contain forward slash `/` in 0.19.0.

## To Reproduce

Configure `SERVER_NAME` or `SERVER_DESCRIPTION` to contain a forward slash `/`, for example a URL and then try to launch container. The name and description will still be default values after starting. 

## Expected behavior

I'd expect to be able to put any sort of character in the server name and description. 

## Logs

Sed error during startup:

```
*****CHECKING FOR EXISTING CONFIG*****
SERVER_NAME="http://palworld.lol 3 | OPEN 24/7 Dedicated"
sed: -e expression #1, char 40: unknown option to `s'
SERVER_DESCRIPTION="Server open to all! More info at http://palworld.lol or https://discord.gg/FYwXY5Ca"
sed: -e expression #1, char 87: unknown option to `s'
```

## docker-compose.yml contents

```yaml
services:
   palworld:
      image: thijsvanloef/palworld-server-docker:latest
      pull_policy: always
      restart: unless-stopped
      container_name: palworld-server
      ports:
        - 8213:8213/udp
        - 27017:27017/udp
      environment:
         - PUID=1000
         - PGID=1000
         - PORT=8213
         - QUERY_PORT=27017
         - PLAYERS=8
         - MULTITHREADING=true
         - RCON_ENABLED=true
         - RCON_PORT=25575
         - ADMIN_PASSWORD="xxxxxxxxxxxxxx"
         - COMMUNITY=true
         - SERVER_NAME="http://palworld.lol 3 | OPEN 24/7 Dedicated"
         - REGION=NA
         - SERVER_DESCRIPTION="Server open to all! More info at http://palworld.lol or https://discord.gg/FYwXY5Ca"
      volumes:
         - ./palworld:/palworld/:z

```

## Additional context

`sed` is interpreting the forward slash inside the server name or description as a separator (e.g. the end of the replacement value) and erroring out.

Here's when I run it manually inside the container:
```
# if [ -n "${SERVER_NAME}" ]; then     echo "SERVER_NAME=${SERVER_NAME}";     sed -E -i "s/ServerName=\"[^\"]*\"/ServerName=$SERVER_NAME/" /palworld/Pal/Saved/Config/LinuxServer/PalWorldSettings.ini; fi

SERVER_NAME="http://palworld.lol 3 | OPEN 24/7 Dedicated"
sed: -e expression #1, char 40: unknown option to `s'
```

https://github.com/thijsvanloef/palworld-server-docker/blob/229d29a5787b44e2973030ceaee407472c981757/scripts/start.sh#L43-L45

**Comment by llnut** [comment]
Perhaps you can use this escape function to process the original server description or server name:
```bash
escape_sed() {
    printf '%s\n' "$1" | sed -e 's:[][\/.^$*]:\\&:g'
}

SERVER_NAME="http://palworld.lol 3 | OPEN 24/7 Dedicated"
sed -E -i "s/ServerName=\"[^\"]*\"/ServerName=\"$(escape_sed "$SERVER_NAME")\"/" /palworld/Pal/Saved/Config/LinuxServer/PalWorldSettings.ini
```

**Comment by thijsvanloef** [comment]
Problem should be resolved in update 0.19.1 thanks to `beomseok-park`, please pull the latest image and check if your issue has been resolved.

**Comment by bostrt** [comment]
Yep! 0.19.1 works great for me. Thanks all.
